tests/cypress/integration/z-newfold-labs/wp-module-onboarding/2-intermediate-steps/0-get-started/TC_05-sitetype-primary.cy.js

```diff
@@ -0,0 +1,109 @@
+// <reference types="Cypress" />
+import { CheckDrawerDisabled } from '../../wp-module-support/drawer.cy';
+import { CheckCardHeadingSubheading } from '../../wp-module-support/header.cy';
+import {
+	CheckHelpPanelLinks,
+	CheckIllustrationPanel,
+	CheckInfoPanel,
+	CheckIntroPanel,
+} from '../../wp-module-support/sidebar.cy';
+
+describe( 'Get Started Site Type Primary', function () {
+	before( () => {
+		cy.visit(
+			'wp-admin/?page=nfd-onboarding#/wp-setup/step/get-started/site-primary'
+		);
+		cy.wait( 3000 );
+	} );
+
+	it( 'Check if the Suppressed Drawer does not open on clicking Toggle Button', () => {
+		CheckDrawerDisabled();
+	} );
+
+	it( 'Check if Header and Subheader shows up', () => {
+		CheckCardHeadingSubheading( true );
+	} );
+
+	it( 'Check to make sure sidebar opens, content is in place and close sidebar', () => {
+		CheckIntroPanel( '__get-started-site-secondary', 'Site Type' );
+		CheckIllustrationPanel();
+		CheckInfoPanel();
+		CheckHelpPanelLinks();
+	} );
+
+	it( 'Check different Categories exist and is selectable', () => {
+		let categoryCount = 0;
+		const className = '.nfd-card-pri-category';
+		cy.get( className ).should( 'be.visible' );
+		const arr = cy.get( className );
+		arr.each( () => {
+			cy.get( className )
+				.eq( categoryCount )
+				.click()
+				.should( 'have.class', 'chosenPrimaryCategory' );
+			categoryCount += 1;
```

Comment:
Should we assert `categoryCount` somewhere later? Otherwise, there is no point of incrementing this?